collections/without_all.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+
+/**
+ * Returns an array excluding all given values.
+ *
+ * Example:
+ *
+ * ```ts
+ * import { withoutAll } from "./without_all.ts";
+ *
+ * const withoutList = withoutAll([2, 1, 2, 3], [1, 2]);
+ *
+ * ```
```

Comment:
I would adapt the example to make it consistent with the others in the module:

```suggestion
 * ```ts
 * import { withoutAll } from "./without_all.ts";
 * import { assertEquals } from "../testing/asserts.ts";
 *
 * const withoutList = withoutAll([2, 1, 2, 3], [1, 2]);
 *
 * assertEquals(withoutList, [3]);
 * ```
```